### ISSUE TITLE
Weird thing on page2

### ISSUE THREAD

**Issue by nitatemic** [nitatemic/AlexandreDeLemeny-Makedone_4_24052021#7]
![image](https://user-images.githubusercontent.com/13944652/119280835-7196e300-bc33-11eb-842f-c4619a942dd9.png)

**Comment by nitatemic** [comment]
I found out what it is. It's the button to bring up the menu that doesn't work anymore
